dags/databricks-ml-example.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+import logging
+
+from airflow.decorators import task, dag
+from airflow.providers.databricks.hooks.databricks import DatabricksHook
+from airflow.providers.databricks.operators.databricks import DatabricksSubmitRunOperator
+# from astronomer.providers.databricks.operators.databricks import DatabricksSubmitRunOperatorAsync
```

Comment:
Is `DatabricksSubmitRunOperatorAsync` really only available to Astronomer customers? It's the `astronomer-providers` package on PyPI that's open-sourced.

Reply:
I got it working but there is something to note... the way I had defined the connection initially (which could have been better) seems to not be compatible with the Async Operator.

Reply:
Due to the xcom push issue going to switch back to the airflow databricks package